src/data/iconData.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
You shouldnt need all of these, I see you didnt use the slice method for the iconData which would explain why all these have been left in